pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -66,9 +69,27 @@ let
   });
   wxGTK-override' = if wxGTK-override == null then wxGTK-prusa else wxGTK-override;
 in
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs:
+let
+  src = fetchFromGitHub {
+    owner = "prusa3d";
+    repo = "PrusaSlicer";
+    hash = "sha256-S0z2v6knkQ+xlABB1zedEGtlxA/65X/vxLh304StfbE=";
+    rev = "version_${finalAttrs.version}";
+  };
+  heatshrink-cmake = heatshrink.overrideAttrs (old: {
```

Comment:
I'm not sure how to deal with such repackaging. Should be another package created, with CMakeLists.txt/Config.cmake.in vendored from prusa-slicer?

Reply:
is this a patch that is being considered for inclusion in heatshrink's upstream? maybe we could put it into the package itself then

otherwise this doesn't look bad. if it's going to stay for longer we may want to move it into a seperate file and then callPackage it from here

Reply:
upstream pr exists https://github.com/atomicobject/heatshrink/pull/77 but upstream hasn't seen any changes since 7 years

Reply:
we may want to switch to a maintained fork

Reply:
this one looks promising: https://github.com/kickmaker/heatshrink

Reply:
Current heatshrink version in nixpkgs is only used by prusa-slicer here, so it is possible to switch to fork
However, switching to [kickmaker/heatshrink](https://github.com/kickmaker/heatshrink) doesn't work out-of-the-box.

I think it would be better to apply prusa's patch (https://github.com/atomicobject/heatshrink/pull/77) on our heatshrink version for now.

Reply:
Added prusa's patch, removed duplicate heatshrink-cmake packages.